doc/source/ray-air/getting-started.rst

```diff
@@ -32,72 +32,103 @@ To get started, install Ray AIR via ``pip install -U "ray[air]"``
 Quick Start
 -----------
 
+Below, we demonstrate how you can use the Ray libraries in a seamless flow
+between distributed frameworks (e.g., XGBoost, Pytorch, and Tensorflow): 
+
+Below, let's start by preprocessing your data with Ray AIR's ``Preprocessors``:
+
 .. tabbed:: XGBoost
 
-    Preprocess your data with a ``Preprocessor``.
 
     .. literalinclude:: doc_code/xgboost_starter.py
         :language: python
         :start-after: __air_xgb_preprocess_start__
         :end-before: __air_xgb_preprocess_end__
 
-    Train a model with an ``XGBoostTrainer``.
+.. tabbed:: Pytorch
 
-    .. literalinclude:: doc_code/xgboost_starter.py
+    .. literalinclude:: doc_code/pytorch_tabular_starter.py
```

Comment:
Can you please fix it so we use identical preprocessing code for all frameworks?

Reply:
Can we separate out the common parts into functions or something like that? Not sure how nicely it will play with `include` here.

Reply:
OK i separated some common parts out.